docs/zdgbook/ObjectPublishing.rst

```diff
@@ -566,25 +566,42 @@ Zope Security
 When using Zope rather than publishing your own modules, the publisher
```

Comment:
Replace `When using Zope rather than publishing your own modules, the` with just `The`